transfers/models.py

```diff
@@ -27,7 +27,7 @@ class UserProfile(models.Model):
     ]
 
     user = models.OneToOneField(User, primary_key=True,
-        on_delete=models.CASCADE)
+        on_delete=models.CASCADE, related_name="userprofile")
```

Comment:
Why is this needed?

Reply:
To get UserProfile from request.user which is needed to get who is logged in currently.

Reply:
You can simply user `user.userprofile`